pkg/storageincentives/agent.go

```diff
@@ -53,7 +53,7 @@ type Monitor interface {
 	IsFullySynced() bool
 }
 
-type sampleData struct {
+type SampleData struct {
```

Comment:
I wouldn't worry about this at this point, since it's still under active development and wont have any significant impact.
 
This was changed because Status struct is already exported and has all values exported. I wanted to follow through this.